app/components/Form/FormBase.tsx

```diff
@@ -1,23 +1,25 @@
-import formTheme from '../../lib/theme/FormWithTheme';
-import { forwardRef } from 'react';
+import defaultTheme from 'lib/theme/DefaultTheme';
+import { forwardRef, useMemo } from 'react';
 import { FormProps, AjvError, withTheme } from '@rjsf/core';
-import { customTransformErrors } from '../../lib/theme/customTransformErrors';
+import { customTransformErrors } from 'lib/theme/customTransformErrors';
 import {
   customFormats,
   customFormatsErrorMessages,
-} from '../../data/jsonSchemaForm/customFormats';
+} from 'data/jsonSchemaForm/customFormats';
 
 interface FormPropsWithTheme<T> extends FormProps<T> {
-  theme?: object;
-  onSubmit: any;
-  formData: any;
+  theme?: any;
 }
 
-const Form = withTheme(formTheme);
 const FormBase: React.ForwardRefRenderFunction<
   object,
   FormPropsWithTheme<object>
 > = (props, ref) => {
+  const Form = useMemo(
```

Comment:
Should we update `CalculationForm` with this?

Reply:
`CalculationForm` is defined as a class component, so it can't use hooks. The `Form` there is defined in the constructor, which is fine.
I'd like it for this `FormBase` component to support calculations one day, in a declarative way that doesn't require branching in the `ApplicationForm` component